buildlib/azure-pipelines.yml

```diff
@@ -51,13 +51,14 @@ stages:
                 | csgrep --mode=grep --invert-match --event "internal warning" --prune-events=1 \
                 > $cppcheck_err
 
-                if [ -s $cppcheck_err ]; then
-                  echo "CppCheck found errors:"
-                  cat $cppcheck_err
-                  exit 100
-                fi
-
+              if [ -s $cppcheck_err ]; then
+                echo "CppCheck found errors:"
+                cat $cppcheck_err
+                echo "##vso[task.logissue type=error]CppCheck found errors"
+                echo "##vso[task.complete result=Failed;]"
```

Comment:
It is better to return 100 and let the infrastructure handle this than to log a result=Failed

Reply:
In this case, AZP reports two errors: `logissue` and `exit 100` which is misleading.